### PR TITLE
Update some guild related packets to use structures

### PR DESCRIPTION
**Pull request by Asheraf**
### Pull Request Prelude

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

File: src/map/packets_struct.h [hunk 1]
```diff
@@ -4058,6 +4058,36 @@ struct PACKET_ZC_C_MARKERINFO {
...
+	uint32 lastLoginTime;
+#endif
+} __attribute__((packed));
+
+struct PACKET_ZC_MEMBERMGR_INFO {
+	int16 PacketType;
+	int16 packetLength;
+	struct GUILD_MEMBER_INFO guildMemberInfo[];
+} __attribute__((packed));
+#if PACKETVER < 20161026
```
Comment (4144) [inline src/map/packets_struct.h:4085]: 0x0aa5 used from 2016-12-14bRagexe and 2016-11-30aRagexeRE

File: src/map/packets_struct.h [hunk 2]
```diff
@@ -4058,6 +4058,36 @@ struct PACKET_ZC_C_MARKERINFO {
 DEFINE_PACKET_HEADER(ZC_C_MARKERINFO, 0x09c1);
 #endif
 
+struct GUILD_MEMBER_INFO {
```
Comment (4144) [inline src/map/packets_struct.h:4061]: better put whole struct under ``#ifdef``

File: src/map/packets_struct.h [hunk 3]
```diff
@@ -4088,6 +4088,53 @@ DEFINE_PACKET_HEADER(ZC_MEMBERMGR_INFO, 0x0154);
...
+	int zeny;
+	int masterGID;
+} __attribute__((packed));
+#endif
+
+#if PACKETVER < 20160622
+//0x150; [4144] this is packet for older versions?
+DEFINE_PACKET_HEADER(ZC_GUILD_INFO, 0x01b6);
+#else
+DEFINE_PACKET_HEADER(ZC_GUILD_INFO, 0x0a84);
```
Comment (4144) [inline src/map/packets_struct.h:4135]: 0x0a84 exists from 2016-10-19aRagexe and 2016-09-21bRagexeRE

File: src/map/packets_struct.h [hunk 4]
```diff
@@ -4088,6 +4088,53 @@ DEFINE_PACKET_HEADER(ZC_MEMBERMGR_INFO, 0x0154);
...
+	int virtue;
+	int emblemVersion;
+	char guildname[NAME_LENGTH];
+	char manageLand[MAP_NAME_LENGTH_EXT];
+	int zeny;
+	int masterGID;
+} __attribute__((packed));
+#endif
+
+#if PACKETVER < 20160622
```
Comment (4144) [inline src/map/packets_struct.h:4131]: packet id probably better put near struct without additional ``#if``

**Review by 4144 — CHANGES_REQUESTED** [review]

**Review by 4144 — APPROVED** [review]

**Review by MishimaHaruna — APPROVED** [review]

**Pull request merged**